plugins/lbaas2/app/javascript/widgets/app/components/healthmonitor/HealthmonitorDetails.jsx

```diff
@@ -114,12 +114,12 @@ const HealthmonitorDetails = ({ loadbalancerID, poolID, healthmonitor }) => {
       <div className="list-entry">
         <div className="row">
           <div className="col-md-12">
-            <b>Retries/Probe Timeout/Interval:</b>
+            <b>Retries/Interval:</b>
           </div>
         </div>
         <div className="row">
           <div className="col-md-12">
-            {healthmonitor.max_retries} / {healthmonitor.timeout} /{" "}
+            {healthmonitor.max_retries} /{" "}
```

Comment:
I'm not sure what the semantics of `/{" "}` are exactly, so please check that my removal is correct, thanks!